webapp/channels/src/utils/post_utils.test.tsx

```diff
@@ -1319,3 +1319,37 @@ describe('PostUtils.getUserOrGroupFromMentionName', () => {
         expect(result).toEqual(expected);
     });
 });
+
+describe('makeGetIsReactionAlreadyAddedToPost', () => {
+    const currentUserId = 'current_user_id';
+
+    const baseState = {
+        entities: {
+            users: {
+                currentUserId,
+            },
+            posts: {
+                reactions: {
+                    post_id_1: {
+                        'current_user_id-smile': {
+                            emoji_name: 'smile',
+                            user_id: currentUserId,
+                            post_id: 'post_id_1',
+                        },
```

Comment:
Just to cover all cases, add another reaction with the `sad` emoji for a different user. That way we make sure the test is also checking the case of "even if the post has that reaction made from another user, the function stills correctly says that the reaction is not yet added by the current user".